.changeset/yellow-lions-behave.md

```diff
@@ -0,0 +1,7 @@
+---
+'@shopify/ui-extensions': minor
+'@shopify/ui-extensions-react': minor
+---
+
+- Adds `purchase.address-autocomplete.suggest` extension target
+- Adds `Sheet` component
```

Comment:
The Sheet component already existed, it is just adding some props.
```suggestion
- Adds the `primaryAction` and `secondaryAction` to the `Sheet` component
```

Reply:
ah okay thanks